server/controllers/following.js

```diff
@@ -0,0 +1,114 @@
+const Following = require("../models/Following");
+const asyncHandler = require("express-async-handler");
+const User = require("../models/User");
+const smartSearch = require("../utils/smartSearch");
+
+// @route GET /following
+// @desc get user's following list
+// @access Private
+exports.getFollowing = asyncHandler(async (req, res, next) => {
+	const data = await Following.findOne(
+		{ userId: req.user.id },
+		{ _id: 0, following: 1 }
+	).populate("following", "_id username image");
+
+	//in case the user has not followed anyone yet
+	if (!data) {
+		res.status(200).json({
+			success: {
+				followingList: [],
+			},
+		});
+	}
+	res.status(200).json({
+		success: {
+			followingList: data.following,
+		},
+	});
+});
+
+// @route GET /following/peopleYouMightKnow
+// @desc returns a list of the peaple the user might know
+// @access Private
+exports.getPeopleYouMightKnow = asyncHandler(async (req, res, next) => {
```

Comment:
nice! In the future we can make this even smarter by including people followed by people you follow